extensions/interactions/TextInput/TextInput.py

```diff
@@ -55,4 +54,15 @@ class TextInput(base.BaseInteraction):
             }]
         },
         'default_value': 1,
+    }, {
+        'name': 'speechRecognitionLanguage',
+        'description': 'Language for speech recognition (only in Chrome).',
+        'schema': {
+            'type': 'unicode',
+            # TODO(sll): Pull '(none)' out into a constant.
```

Comment:
This should default to the language of the exploration (but be not restricted to it -- e.g. an exploration that's written in English may want to teach saying basic words in Spanish). Perhaps add a TODO.